CONTRIBUTING.md

```diff
@@ -6,12 +6,12 @@ Thank you!
 
 ## Legal Info
 
-When you open your first pull-request to ClickHouse repo, a bot will invite you to accept ClickHouse Individual CLA (Contributor License Agreement). It is a simple few click process. For subsequent pull-requests the bot will check if you have already signed it and won't bother you again.
+When you create your first pull request to the ClickHouse repository, a bot will invite you to accept the ClickHouse Individual CLA (Contributor License Agreement). It is a simple, few-click process. For subsequent pull requests, the bot will check if you have already signed them and won't bother you again.
 
-Optionally, to make contributions even more tight legally, your employer as a legal entity may want to sign a ClickHouse Corporate CLA with ClickHouse, Inc. If you're interested to do so, contact us at [legal@clickhouse.com](mailto:legal@clickhouse.com).
+Optionally, to make contributions legally tighter, your employer as a legal entity may want to sign a ClickHouse Corporate CLA with ClickHouse, Inc. If you're interested, contact us at [legal@clickhouse.com](mailto:legal@clickhouse.com).
```

Comment:
While many automated tools suggest "tighter", it's ok to write "even more tight" as a stylistic option.